lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -2061,7 +2079,7 @@ sub done {
         # limit length of the reason
         # note: The reason can be anything the worker picked up as useful information so better cut it at a
         #       reasonable, human-readable length. This also avoids growing the database too big.
-        $reason = substr($reason, 0, 300) . decode('UTF-8', '…') if defined $reason && length $reason > 300;
+        $reason = substr($reason, 0, 300) . '…' if defined $reason && length $reason > 300;
```

Comment:
interesting. Why this change?

Reply:
see https://github.com/os-autoinst/openQA/pull/3859